iron-doc-viewer.html

```diff
@@ -119,6 +119,22 @@
         observer: '_descriptorChanged',
       },
 
+      /**
+       * Set to true when running in the catalog.
+       *
+       * By default, hash link includes the component name,
+       * so links work when an `iron-component-page` is
+       * served statically. In the catalog, the component name
+       * is omitted because it's already included in a URL parameter.
+       *
+       * In the catalog, the app handles routing &
+       * scrolling to the selected anchor.
+       */
+      catalog: {
```

Comment:
I had originally named it `useLocalAnchors`, in an attempt to make it more generic, but in practice, I think it's code that's only going to be used in the catalog, so I thought maybe explicit was better.